### ISSUE TITLE
[JOSS REVIEW] Run your code through a linter

### ISSUE THREAD

**Issue by finsberg** [florisvanvugt/biobabel#6]
There are a lot of unused imports and code blocks that don't follow the standard [PEP8](https://peps.python.org/pep-0008/) style guide for python. I would recommend to run a linter and formatter on all your code and fix the issues that are pointed out (unless you have very strong opinions on style, in which case you should write this in the CONTRIBUTING guide).

A popular linting and formatting tool that you can use is [ruff](https://docs.astral.sh/ruff/ )

Note that that this is not a requirement but rather a recommendation.

This is related to https://github.com/openjournals/joss-reviews/issues/6884

**Comment by sappelhoff** [comment]
I agree with everything @finsberg says. I would furthermore recommend you adopt a particular style for docstrings (and use it consistently), and that you add [module docstrings](https://stackoverflow.com/q/2557110/5201771) to each `.py` file. I recommend [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html).
